arch/power/power.c

```diff
@@ -0,0 +1,24 @@
+/* POWER feature check
+ * Copyright (C) 2020 Matheus Castanho <msc@linux.ibm.com>, IBM
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#include <sys/auxv.h>
+#include "../../zutil.h"
```

Comment:
After fixing SRCTOP in Makefile.in, this should be changed to "zutil.h" without path...

Reply:
Ok. Queued for next push.